broker/setup_broker/lvk/setup_broker.sh

```diff
@@ -34,15 +34,31 @@ fi
 
 #--- GCP resources used directly in this script
 broker_bucket="${PROJECT_ID}-${survey}-broker_files"
+bq_dataset="${survey}"
 topic_alerts="${survey}-alerts"
+topic_storebigquery="${survey}-bigquery"
+
 # use test resources, if requested
 if [ "$testid" != "False" ]; then
     broker_bucket="${broker_bucket}-${testid}"
+    bq_dataset="${bq_dataset}_${testid}"
     topic_alerts="${topic_alerts}-${testid}"
+    topic_storebigquery="${topic_storebigquery}-${testid}"
 fi
 
-#--- Create (or delete) GCS, Pub/Sub resources
+alerts_table="alerts"
+
+#--- Create (or delete) BigQuery, GCS, Pub/Sub resources
+echo
+echo "Configuring BigQuery, GCS, Pub/Sub resources..."
 if [ "${teardown}" != "True" ]; then
+    # create bigquery dataset and table
+    bq --location="${region}" mk --dataset "${bq_dataset}"
+
+    cd templates || exit
```

Comment:
should have an exit code (and ideally, helpful message) to debug if an issue occurs.